### ISSUE TITLE
can not import github.com/tensorflow/tensorflow/tensorflow/go

### ISSUE THREAD

**Issue by mattn** [gopherdata/gophernotes#172]
![image](https://user-images.githubusercontent.com/10111/58748885-9e3b1880-84b9-11e9-9b0b-ee768705a58f.png)

```
repl.go:4:5: error executing "go build -buildmode=plugin" in directory "/home/mattn/go/src/gomacro_imports/github.com/tensorflow/tensorflow/tensorflow/go/": exit status 1
```

```
$ cd /home/mattn/go/src/gomacro_imports/github.com/tensorflow/tensorflow/tensorflow/go/
$ go build -buildmode=plugin
can't load package: package gomacro_imports/github.com/tensorflow/tensorflow/tensorflow/go:
go.go:8:2: expected 'STRING', found 'go'
go.go:10:2: expected ')', found 'EOF'
```

```
$ head go.go
// this file was generated by gomacro command: import "github.com/tensorflow/tensorflow/tensorflow/go"
// DO NOT EDIT! Any change will be lost when the file is re-generated

package main

import (
        . "reflect"
        go "github.com/tensorflow/tensorflow/tensorflow/go"
)
```

**Comment by cosmos72** [comment]
Ops... good catch!
the importer does not check yet for reserved keywords and attempts to use `go` as package import alias.
I will fix it ASAP

**Comment by cosmos72** [comment]
fixed in 7c3ca4e0f5c3a1a43c659f86e3cd02a9a9b9aa9a
